politeiawww/inventory.go

```diff
@@ -224,6 +225,45 @@ func (b *backend) loadRecord(record pd.Record) error {
 		return fmt.Errorf("could not load comments for %s: %v", t, err)
 	}
 
+	err = b.loadCommentsLikes(t)
+	if err != nil {
+		return fmt.Errorf("could not load comment likes for %s: %v", t, err)
+	}
+
+	err = b.processCommentsLikesResults(t)
+	if err != nil {
+		return fmt.Errorf("could not process comment likes results for %s: %v", t, err)
+	}
+	// b.calculateCommentsLikesScore(t)
+
+	return nil
+}
+
+// processCommentsLikesResults calculates the total votes and the result
+// votes for each comment of a given proposal. It also calculates the
+// resultant action per user per comment
+//
+// This function must be called WITH the mutext held
+func (b *backend) processCommentsLikesResults(token string) error {
```

Comment:
If this needs to be called with the mutex held it should be named `_processCommentsLikesResults`, right?